modules/appeals_api/spec/docs/v2/sc_spec.rb

```diff
@@ -159,6 +159,189 @@
+
+        before do |example|
+          submit_request(example.metadata)
+        end
+
+        after do |example|
+          response_title = example.metadata[:description]
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              examples: {
+                "#{response_title}": {
+                  value: JSON.parse(response.body, symbolize_names: true)
+                }
+              }
+            }
+          }
+        end
+
+        it 'minimum fields used' do |example|
+          assert_response_matches_metadata(example.metadata)
+        end
+      end
+
+      response '200', 'Valid maximum' do
+        let(:sc_body) do
+          JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'fixtures', 'v2', 'valid_200995_extra.json'))).tap do |data|
+            unless DocHelpers.wip_doc_enabled?(:sc_v2_claimant)
+              data.dig('data', 'attributes')&.delete('claimant')
+              data.dig('data', 'attributes')&.delete('claimantTypeOtherValue')
+              data['data']['attributes']['claimantType'] = 'veteran'
+            end
+          end
+        end
+
+        if DocHelpers.wip_doc_enabled?(:sc_v2_claimant)
+          let(:'X-VA-Claimant-First-Name') { 'first' }
+          let(:'X-VA-Claimant-Last-Name') { 'last' }
+        end
+
+        schema JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'support', 'schemas', 'sc_validate.json')))
+
+        before do |example|
+          submit_request(example.metadata)
+        end
+
+        after do |example|
+          response_title = example.metadata[:description]
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              examples: {
+                "#{response_title}": {
+                  value: JSON.parse(response.body, symbolize_names: true)
+                }
+              }
+            }
+          }
+        end
+
+        it 'all fields used' do |example|
+          assert_response_matches_metadata(example.metadata)
+        end
+      end
+
+      response '422', 'Violates JSON schema' do
+        schema '$ref' => '#/components/schemas/errorModel'
+
+        let(:sc_body) do
+          request_body = JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'fixtures', 'v2', 'valid_200995.json')))
+          request_body['data']['attributes'].delete('veteran')
+          request_body
+        end
+
+        before do |example|
+          submit_request(example.metadata)
+        end
+
+        after do |example|
+          response_title = example.metadata[:description]
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              examples: {
+                "#{response_title}": {
+                  value: JSON.parse(response.body, symbolize_names: true)
+                }
+              }
+            }
+          }
+        end
+
+        it_behaves_like 'rswag example', desc: 'returns a 422 response'
```

Comment:
You shouldn't need the `before` and `after` when using the `rswag example` since it takes care of that setup for you.